11: Delete empty pytest.ini

EugeneTurkin opened the pull request:
Parent story: https://github.com/fenya123/bingin/issues/10

During #7 we've made a mistake of leaving an empty `pytest.ini` to lay around. No sense in having an empty file that does nothing.

In the scope of this task we will get rid of `pytest.ini`.

birthdaysgift commented:
Since this task can be done separately from #10, let's rebase this branch (`11-delete-empty-pytest-ini`) to the develop, so these two branches will be independent. Because for now if you'll look at the PR then you see that it includes all changes from `10-add-linters-to-ci`.

EugeneTurkin committed:
11: Delete empty pytest.ini

Parent story: https://github.com/fenya123/bingin/issues/10

During #7 we've made a mistake of leaving an empty pytest.ini to lay around. No sense in having an empty file that does nothing.

In the scope of this task we will get rid of pytest.ini.
